app/services/data_export/taxon_export.rb

```diff
@@ -1,10 +1,8 @@
 module DataExport
   class TaxonExport
-    TAXON_FIELDS = %w[content_id base_path title].freeze
```

Comment:
I'm not sure what benefit we're getting having removed this constant. Instead, we're now passing down  `content_id` and `base_path` in almost every method in `TaxonomyQuery` which I think is causing unnessary noise. That is unless I've massively overlooked something!